provisioning/ubuntu-provision.sh

```diff
@@ -532,6 +532,14 @@ function install_goss() {
   chmod +rx /usr/local/bin/goss
 }
 
+function install_tfsec() {
+  apt-get update --quiet
+  apt-get install --yes --no-install-recommends curl # Should already be there but this function should be autonomous
+
+  curl --silent --location --show-error "https://github.com/aquasecurity/tfsec/releases/download/v${TFSEC_VERSION}/tfsec-checkgen-linux-${ARCHITECTURE}" --output /usr/local/bin/tfsec
```

Comment:
```suggestion
  curl --silent --location --show-error "https://github.com/aquasecurity/tfsec/releases/download/v${TFSEC_VERSION}/tfsec-linux-${ARCHITECTURE}" --output /usr/local/bin/tfsec
```

I think the fact only tfsec-checkgen release assets are displayed by default induced you in error:
<img width="780" alt="image" src="https://user-images.githubusercontent.com/91831478/211300048-3f3fa329-921e-407c-b397-ecda13a3138e.png">

Reply:
Yeah indeed, you're right 😅 

Sorry about that and thanks for the fix!